db/model/Gdoc/rawToEnriched.ts

```diff
@@ -1879,3 +1884,61 @@ export function parseRefs({
 
     return { definitions: parsedRefs, errors: refErrors }
 }
+
+const parseKeyIndicator = (
+    raw: RawBlockKeyIndicator
+): EnrichedBlockKeyIndicator => {
+    const createError = (
+        error: ParseError,
+        datapageUrl = ""
+    ): EnrichedBlockKeyIndicator => ({
+        type: "key-indicator",
+        datapageUrl,
+        title: "",
+        blurb: [],
+        parseErrors: [error],
+    })
+
+    const val = raw.value
+
+    if (typeof val === "string")
+        return createError({
+            message: "Value is a string, not an object with properties",
+        })
```

Comment:
This error seems quite unlikely (I've never been asked about it in the last ~year of gdocs) but if it were to happen, I think the message probably wouldn't make much sense to the authors. Either including the correct archie syntax in the message or directing them to the documentation might be helpful.